confiture-web-app/src/components/audit/NotesModal.vue

```diff
@@ -19,97 +31,199 @@ defineExpose({
   hide: () => modal.value?.hide()
 });
 
+const showFileSizeError = ref(false);
+const showFileFormatError = ref(false);
+
 const auditStore = useAuditStore();
+const notify = useNotifications();
+const route = useRoute();
 
 const modal = ref<InstanceType<typeof DsfrModal>>();
 const isOffline = useIsOffline();
 
-const notes = ref("");
+const notes = ref(auditStore.currentAudit?.notes || "");
 
-function handleSubmit() {
-  emit("confirm", notes.value);
-}
+const uniqueId = computed(() => route.params.uniqueId as string);
+const files = computed(() => auditStore.currentAudit?.notesFiles || []);
 
-function handleClose() {
-  modal.value?.hide();
+const handleNotesChange = debounce(() => emit("confirm", notes.value), 500);
+
+function handleUploadFile(file: File) {
+  showFileSizeError.value = false;
+  showFileFormatError.value = false;
+  notify("info", "Chargement en cours");
```

Comment:
Je crois que @AdrienMuzyczka voulait harmoniser les toasts et notamment en mettant plutôt juste la description plutôt que le titre car certains ont juste un titre, d'autres juste une description...

```suggestion
  notify("info", undefined, "Chargement en cours");
```

Reply:
Notification supprimée… mais je note pour les prochaines.